R/plotSeries.R

```diff
@@ -0,0 +1,147 @@
+#' Plot Series
+#'
+#' Function that plots core taxa against time. #####################################
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object.
+#'
+#' @param abund_values
+#' A single character value for selecting the
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{assay}}
+#'   to be plotted.
+#'
+#' @param x_axis
+#'  A single character value for selecting the column from 
+#'  \code{\link[SummarizedExperiment:SummarizedExperiment-class]{ColData}} 
+#'  that will specify values of x-axis. 
+#'  
+#' @param x_axis
+#'  A single character value for selecting the taxa from 
+#'  \code{\link[SummarizedExperiment:SummarizedExperiment-class]{rownames}}. 
+#'  This parameter specifies taxa whose abundances will be plotted.
+#'  
+#'  @param rank a single character defining a taxonomic rank, that is used to
+#'  agglomerate the data. Must be a value of \code{taxonomicRanks()} function.
+#'
+#' @details
+#' Creates a plot where core taxa is presented against time. ###############################
+#'
+#'
+#' @references
+#' Add here.#####################################
+#'
+#' @return
+#' Returns plot###############################
+#'
+#' @name plotSeries
+#' @export
+#'
+#' @author Leo Lahti and Tuomas Borman. Contact: \url{microbiome.github.io}
+#'
+#' @examples
+#' data(esophagus)
+#' x <- esophagus
+#'
+NULL
+
+#' @rdname plotSeries
+#' @export
+setGeneric("plotSeries", signature = c("x"),
+           function(x,
+                    abund_values = NULL,
+                    x_axis = NULL,
+                    y_axis = NULL,
+                    rank = NULL)
```

Comment:
I think you can remove the specific arguments except for `object`. (Use `object` instead of `x` so you avoid the clash and the argument names are the same as for the other plotting functions)